keyboards/handwired/yamsek/atmega32u4/rules.mk

```diff
@@ -0,0 +1,19 @@
+# MCU name
+MCU = atmega32u4
+BOOTLOADER = caterina
+
+RGBLIGHT_ENABLE = yes
+RGBLIGHT_DRIVER = WS2812
+
+TRACKBALL_ENABLE = no
+POINTING_DEVICE_ENABLE = no
+POINTING_DEVICE_DRIVER = pimoroni_trackball
+
+## saving space to squeeze in VIAL support
+AUTO_SHIFT_ENABLE = no
+COMBO_ENABLE = no
+#DEBUG_ENABLE = no
+#  KEY_OVERRIDE_ENABLE            = yes              # file
+#  OLED_DRIVER                    = SSD1306          # file
+#  RAW_ENABLE                     = yes              # file
+TAP_DANCE_ENABLE = no
```

Comment:
You're not saving much space by explicitly disabling features that are already off by default.

Reply:
the "default" sadly changes with other forks/ports... that enable $things through builddefs :-/
this way the unused parts are "forced" off

thanks for the hint - i'll do some cleanup

Reply:
added a comment, which hopefully clarifies the intent